lib/node_modules/@stdlib/math/base/special/sincos/test/fixtures/julia/runner.jl

```diff
@@ -0,0 +1,69 @@
+#!/usr/bin/env julia
+
+import JSON
+
+"""
+    gen( x, filepath )
+
+Generate fixture data and write to file.
+
+# Arguments
+
+* `x`: domain
+* `filepath::AbstractString`: filepath of the output file
+
+# Examples
+
+``` julia
+julia> x = linspace( -1000.0, 1000.0, 2001 );
+julia> gen( x, \"./data.json\" );
+```
+"""
+function gen( x, filepath )
+	s = sin( x );
+	c = cos( x );
+	data = Dict([
+		("x", x),
+		("sine", s),
+		("cosine", c)
+	]);
+	outfile = open( filepath, "w" );
+	write( outfile, JSON.json(data) );
+	close( outfile );
+end
+
+# Get the filename:
+file = @__FILE__;
+
+# Extract the directory in which this file resides:
+dir = dirname( file );
+
+# Negative medium sized values:
+x = linspace( -256.0*pi, 0.0, 4000 )
+out = joinpath( dir, "medium_negative.json" );
+gen( x, out );
+
+# Positive medium sized values:
+x = linspace( 0.0, 256.0*pi, 4000 )
```

Comment:
Unfortunately, the Julia linter (https://lintjl.readthedocs.io/en/stable/messages/) is not as configurable as ESLint and, in fact, not configurable at all. So we are left manually enforcing these kinds of things.